### PR TITLE
Clean up design for messageboards index and new

### PR DESCRIPTION
**Pull request by kylefiedler**
screenshot 2015-06-09 21 32 38
screenshot 2015-06-09 21 32 50

### CODE REVIEW

File: spec/dummy/app/themes/default/assets/stylesheets/components/_form-list.scss [hunk 1]
```diff
@@ -23,3 +23,12 @@
...
   }
 }
+
+.form-list--hint {
+  color: lighten($base-font-color, 20%);
+  display: inline-block;
+  font-size: rem(14);
+  position: relative;
+  top: -($small-spacing / 2);
+  margin-bottom: $small-spacing;
```
Comment (kylefiedler) [inline spec/dummy/app/themes/default/assets/stylesheets/components/_form-list.scss]: :abc:

Comment (kylefiedler) [inline spec/dummy/app/themes/default/assets/stylesheets/components/_form-list.scss]: @jayroh any interest in adding hound to this project? https://houndci.com/

Comment (jayroh) [inline spec/dummy/app/themes/default/assets/stylesheets/components/_form-list.scss]: Know what - I think that's a great idea. Free for open source projects, right?

**Comment by jayroh** [comment]
This closed after merging `kf-topics-index` but I have since cherry-picked the commit. Thank you @kylefiedler!

**Pull request closed without merging**